ISSUE_TEMPLATE/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+Issue tracker is **ONLY** used for reporting bugs. Support or new features should be discussed on [Lava's Discord channel](https://discord.gg/5VcqgwMmkA).
```

Comment:
Consider that by asking users that are on your Github to go somewhere else, you could be losing out on their feedback.

Reply:
Thanks, it's an interim time until we fully accept all contributions.